browser/model/helpers/installer.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+let fs = require('fs-extra');
+let child_process = require('child_process');
+let unzip = require('unzip');
+
+import Logger from '../../services/logger';
+
+class Installer {
+
+  constructor(key, progress, success, failure) {
+    this.progress = progress;
+    this.success = success;
+    this.failure = failure;
+    this.key = key;
+  }
+
+  exec(command, options, result) {
+    let self = this;
```

Comment:
Do you mean defining exec(command, options, result) using arrow syntax? Afaik that is not allowed.